include/bluetooth/mesh/light_ctl_srv.rst

```diff
@@ -0,0 +1,128 @@
+.. _bt_mesh_light_ctl_srv_readme:
+
+Light CTL Server
+######################
+
+The Light Color Tunable Light (CTL) Server represents a single light on a mesh device.
+It should be instantiated in the light fixture node.
+
+The Light CTL Server adds the following new model instances in the composition data, in addition to the extended :ref:`bt_mesh_lightness_srv_readme` model:
+
+* Light CTL Server
+* Light CTL Setup Server
+
+These model instances share the states of Light CTL Server, but accept different messages.
+This allows for a fine-grained control of the access rights for the Light CTL states, as these model instances can be bound to different application keys:
+
+* The Light CTL Server provides write access to the Lightness, Temperature and Delta UV states for the user, in addition to read access to all the meta states.
+* The Light CTL Setup Server provides write access to the Default CTL state and Temperature Range meta states.
+  This allows the configurator devices to set up the Temperature range and the default CTL state.
+
+In addition to the extended Lightness Server model the Light CTL Server is also assosiated with an instance of the :ref:`bt_mesh_light_temp_srv_readme` model.
+This model instance must be located on the same device, where the element address of the Light Temperature Server must be greater than the address of the
+Light CTL Server.
+
+States
+======
+
+The Generic Power OnOff Server model (extended by the Lightness Server model) contains the following states:
+
+Lightness: ``uint16_t``
+    The Lightness state represents the emitted light level of an element, and ranges from ``0`` to ``65535``.
+    The Lightness state is bound to the Light Lightness Actual state of the extended :ref:`bt_mesh_lightness_srv_readme`:
+
+    .. code-block:: console
+
+       Light CTL Lightness = Light Lightness Actual
```

Comment:
This isn't quite true. If the lightness representation is linear, the CTL lightness will be linear as well. Better just to say that the CTL Server's Lightness state is shared with the extended Lightness Server's Lightness state.